src/components/PlayersList.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { connect } from 'react-redux';
+// import fetchPlayersReducer from '../reducers/fetchPlayersReducer.js'
+import { fetchPlayers } from '../actions/actions.js';
+import { selectPlayer } from '../actions/actions.js';
+
+class PlayersList extends React.Component {
+    componentDidMount() {
+        this.props.fetchPlayers();
+    }
+
+    componentDidUpdate() {
```

Comment:
No need to put this into componentDidUpdate. This is not the correct usage of `componentDidUpdate`. This code should move to the render function below. See render function comment.

Reply:
I have renamed this component renderList( ). I've been told to keep the render method as clean as possible by devs and in videos (and I prefer it that method).